data-library/asthma/maintenance.cfg

```diff
@@ -0,0 +1,22 @@
+############
+# if an op is supported, add an {op}_interval section with the appropriate interval values
+# for now all values are mandatory. Example:
+#   days = 2
+#   weeks = 0
+#   months = 2
+#   years = 0
+# this means the op should be run every 2 months and 2 days.
+# if an op is not supported by the current pipeline, simply omit the section and it will be ignored.
+############
+
+[fetch_interval]
+days = 0
+weeks = 0
+months = 6
+years = 0
+
+[clean_interval]
+days = 1
+weeks = 0
+months = 0
+years = 0
```

Comment:
I actually don't think we'll need to keep a separate interval for cleaning - the only times we'll need to clean will be: 1) immediately after we fetch and 2) any time we make changes to the pipeline's clean script. We only need the fetch interval to cover the first case, and the second case will just require manual cleaning, since it's a consequence of manual changes. So, I think the only things we'll need to track in a maintenance file would be: 1) the fetch interval, 2) the last time fetch executed, and 3) the last time clean executed.